src/CSModule.sol

```diff
@@ -1102,9 +1104,15 @@ contract CSModule is
 
         emit WithdrawalSubmitted(nodeOperatorId, keyIndex, amount);
 
-        if (_isValidatorSlashed[pointer]) {
-            unchecked {
-                amount += INITIAL_SLASHING_PENALTY;
+        if (isSlashed) {
+            if (_isValidatorSlashed[pointer]) {
+                // Account already burned value
+                unchecked {
+                    amount += INITIAL_SLASHING_PENALTY;
+                }
+            } else {
+                _isValidatorSlashed[pointer] = true;
+                emit InitialSlashingSubmitted(nodeOperatorId, keyIndex);
```

Comment:
It doesn't make sense to emit this event and to update the `_isValidatorSlashed` (we won't read it again). I think about something like `_isValidatorSlashed -> _initialSlashingSubmitted`. So we will have this exact case clearly separated.